src/IO/H5/Python/VolumeData.cpp

```diff
@@ -0,0 +1,71 @@
+
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <Python.h>
+#include <boost/python.hpp>
+#include <boost/python/stl_iterator.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+#include <hdf5.h>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "IO/H5/Header.hpp"
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+#include "IO/H5/Python/Numpy.hpp"
+#include "IO/H5/Python/ToNumpy.hpp"
+#include "IO/H5/Version.hpp"
+#include "IO/H5/VolumeData.hpp"
+#include "PythonBindings/VectorPyList.hpp"
+
+namespace bp = boost::python;
+
+namespace py_bindings {
+void bind_h5vol() {
+  // Wrapper for basic H5VolumeData operations
+  bp::class_<h5::VolumeData, boost::noncopyable>("H5Vol", bp::no_init)
+      .def("extension", &h5::VolumeData::extension)
+      // The method extension() is static
+      .staticmethod("extension")
+
+      .def("get_header", +[](h5::VolumeData& V) { return V.get_header(); })
+
+      .def("get_version", +[](h5::VolumeData& V) { return V.get_version(); })
```

Comment:
for these `get` functions can the `h5::VolumeData` be a `const&`? If not, it would be useful to add a comment right before `get_header` stating why.